src/test/java/org/jenkinsci/plugins/gitclient/GitAPITestCase.java

```diff
@@ -178,40 +171,37 @@ private void assertSubstringTimeout(final String substring, int expectedTimeout)
         }
 
         private void setupProxy(GitClient gitClient)
-              throws SecurityException, NoSuchFieldException, IllegalArgumentException, IllegalAccessException
-        {
-          final String proxyHost = getSystemProperty("proxyHost", "http.proxyHost", "https.proxyHost");
-          final String proxyPort = getSystemProperty("proxyPort", "http.proxyPort", "https.proxyPort");
-          final String proxyUser = getSystemProperty("proxyUser", "http.proxyUser", "https.proxyUser");
-          //final String proxyPassword = getSystemProperty("proxyPassword", "http.proxyPassword", "https.proxyPassword");
-          final String noProxyHosts = getSystemProperty("noProxyHosts", "http.noProxyHosts", "https.noProxyHosts");
-          if(isBlank(proxyHost) || isBlank(proxyPort)) return;
-          ProxyConfiguration proxyConfig = new ObjenesisStd().newInstance(ProxyConfiguration.class);
-          setField(ProxyConfiguration.class, "name", proxyConfig, proxyHost);
-          setField(ProxyConfiguration.class, "port", proxyConfig, Integer.parseInt(proxyPort));
-          setField(ProxyConfiguration.class, "userName", proxyConfig, proxyUser);
-          setField(ProxyConfiguration.class, "noProxyHost", proxyConfig, noProxyHosts);
-          //Password does not work since a set password results in a "Secret" call which expects a running Jenkins
-          setField(ProxyConfiguration.class, "password", proxyConfig, null);
-          setField(ProxyConfiguration.class, "secretPassword", proxyConfig, null);
-          gitClient.setProxy(proxyConfig);
+                throws SecurityException, NoSuchFieldException, IllegalArgumentException, IllegalAccessException {
```

Comment:
Please don't perform wholesale reformatting without discussion with the maintainers

Reply:
Could you have a short look at the two individual "reformatting" commits 42305b580ecd657d9bfec950716296d0b5691117 and 6b4f392d9d1eb309748c613017eccb0ba5b4eec9 and provide feedback here if to remove the commits completely or if some changes are welcome?